src/app/pages/schedule-filter/schedule-filter.html

```diff
@@ -19,11 +19,12 @@
   <ion-list [lines]="ios ? 'inset' : 'full'">
     <ion-list-header>Tracks</ion-list-header>
 
-    <ion-item *ngFor="let track of tracks" [attr.track]="track.name | lowercase">
-      <ion-icon *ngIf="ios" slot="start" [name]="track.icon" color="medium"></ion-icon>
-      <ion-label>{{track.name}}</ion-label>
-      <ion-checkbox [(ngModel)]="track.isChecked"></ion-checkbox>
-    </ion-item>
+<ion-item *ngFor="let track of tracks" [attr.track]="track.name | lowercase">
+  <ion-icon *ngIf="ios" slot="start" [name]="track.icon" color="medium"></ion-icon>
+  <ion-checkbox [(ngModel)]="track.isChecked">
+    {{track.name}}
+  </ion-checkbox>
+</ion-item>
```

Comment:
The filters also no longer seem to be working. On the `master` branch, I can uncheck categories and it'll actually filter those out, but all events always display when on the branch.

Reply:
This was caused by [removing the `hidden` attribute CSS](https://github.com/ionic-team/ionic-framework/blob/feature-7.0/BREAKING.md#hidden-attribute). Should we move this into the migration guide? It seems like something that is easy to overlook.

Reply:
The migration guide should be focused on changes that are required in order for your app to compile. Perhaps we could move this change to the top of the file?

Reply:
I wouldn't be opposed to it 👀

Reply:
Sure CSS Utilities works - want to open a PR?

Reply:
Yeah I can do that!